src/coreclr/jit/compiler.h

```diff
@@ -2833,6 +2831,19 @@ class Compiler
 
     GenTreeIndir* gtNewIndir(var_types typ, GenTree* addr, GenTreeFlags indirFlags = GTF_EMPTY);
 
+    GenTree* gtNewLoadValueNode(
```

Comment:
Do you plan on a similar `gtNewStoreValueNode` for parity?

Right now that's one of a few nodes, including `ASG` iirc?

Reply:
> Do you plan on a similar `gtNewStoreValueNode` for parity?

Yep, these are all part of the `ASG` removal project.